### PR TITLE
Update "Create EHR" tests

### PR DESCRIPTION
**Pull request by testautomation**
## Changes

- Changes status codes according to updated test docs
  as mentined in https://github.com/ehrbase/project_management/issues/154#issuecomment-610572942
- moves a data set into proper (invalid) subfolder

closes https://github.com/ehrbase/project_management/issues/154

### CODE REVIEW

**Comment by testautomation** [comment]
@jakesmolka due to changes applied some tests in EHR_SERIVE tests suite are failing now (this was expected). As mentioned in https://github.com/ehrbase/project_management/issues/154#issuecomment-610391280 now it's your turn to

> remove json preprocessing

Please push your changes into this PR to make these test PASS again.

**Comment by testautomation** [comment]
I set this PR to DRAFT to make sure it is not merged before https://github.com/ehrbase/ehrbase/pull/225 was reviewed and merged.

**Comment by jakesmolka** [comment]
I could fix the issues marked with ehrbase/project_management#154
- 014, 015, 016
- 046, 047, 048

But all other failing ones that are not marked as not-ready have some problems
- 018 `tests/robot/_resources/test_data_sets/ehr/invalid/001_ehr_status_subject_empty.json` is a valid scenario. Subject is 1..1 but the content has 0..1.
- 023 seems broken as it should fail but then appears to look for a valid response `Expected property 'response body ehr_status is_modifiable' was not found.`. Fixed some sub-cases, but this test needs to be re-evaluated by me again.
- 024 seems to have the same problem
- 030 same
- 031 is like 018
- (032 might to be off too, but didn't looked into it, as it just turned up)
- 050 uses the same test file like 018

But this is based on the local tests. Lets seem what the CI says.

**Comment by testautomation** [comment]
@jakesmolka 

- > 018 tests/robot/_resources/test_data_sets/ehr/invalid/001_ehr_status_subject_empty.json is a 
valid scenario. Subject is 1..1 but the content has 0..1.

  providing an empty JSON `{}`as subject **was** a valid scenario, since changes in test docs it's considered invalid

- > 023 seems broken ...

  indeed, I'v fixed it by separating valid an invalid cases. MF-023 now has only valid cases, MF-024 has the invalid cases

- > 024

  consolidated w/ invalid cases in 025

- > 030 seems broken

  yes, fixed by consolidated invalid cases w/ 031

- > 031

  same as in 018: the variants are failing where subject is provided as empty JSON `{}`

- > 032

  seems the order of validation has changed. The test checkes that where is an error message related to invalid ehr_id. This works for the first two variants but not for the rest where error messages is related to invalid attribute `invalid: EHR_STATUS is_modifiable' is not one of ['EHR ID format not a UUID']`. This is something that could be discussed but I simply removed the error message check so the test PASSes now.

- > 050

  same as 018, is the new invalid case, was valid formerly


To sum up: what's left to be fixed is to treat a subject that is provided as empty JSON `{}` as invalid.
I've marked related tests properly with 154 tag, you can rerun them with below command - that should give you 4 failing tests (018, 025, 031, 050): 

```
robot -i 154 -d results -L TRACE -v nodocker robot/EHR_SERVICE_TESTS
```
